### ISSUE TITLE
Module Manifest Template Tests Don't Work with Pester 5

### ISSUE THREAD

**Issue by mattmcnabb** [PowerShellOrg/Plaster#363]
The built-in module manifest template includes this test:

```powershell
It 'Passes Test-ModuleManifest' {
        Test-ModuleManifest -Path $ModuleManifestPath | Should Not BeNullOrEmpty
        $? | Should Be $true
    }
```

Which returns this error with Pester 5:

```
Should : Legacy Should syntax (without dashes) is not supported in Pester 5. Please refer to migration guide at: https://pester.dev/docs/migrations/v3-to-v4
At C:\Users\matt\Documents\Repos\JsonWhois\test\JsonWhois.Tests.ps1:6 char:57
+ ... t-ModuleManifest -Path $ModuleManifestPath | Should Not BeNullOrEmpty
+                                                  ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Should
```

Can we update this to use the canonical `Should -Not -BeNullOrEmpty` instead of `Should Not BeNullOrEmpty`? Thanks!

**Comment by spuninst** [comment]
7th line needs updating as well. >  $? | Should -Be $true <  Thanks.

**Comment by HeyItsGilbert** [comment]
This should be resolved with #419.

**Comment by ergo3114** [comment]
Tests are completing successfully on the latest build #420. Closing issues as tests have been fixed for pester 5.
